### PR TITLE
New package: linux-vt-setcolors-1.0.0

### PR DESCRIPTION
**Pull request by SevereOverfl0w**

### CODE REVIEW

File: srcpkgs/linux-vt-setcolors/patches/8.diff [hunk 1]
```diff
@@ -0,0 +1,15 @@
+diff --git Makefile Makefile
+index 7a9cc3e..ad5ad1d 100644
+--- Makefile
++++ Makefile
+@@ -1,7 +1,9 @@
+ PREFIX=/usr/local
++CC=gcc
```
Comment (ericonr) [inline srcpkgs/linux-vt-setcolors/patches/8.diff:7]: You don't need to specify `CC`, `make` can decide on one automatically.

**Review by ericonr — COMMENTED** [review]
Are you still interested in getting this package merged?

File: srcpkgs/linux-vt-setcolors/patches/8.diff (hunk 1 above)
Comment (SevereOverfl0w) [reply srcpkgs/linux-vt-setcolors/patches/8.diff:7]: Ah, my upstream patch was apparently a little wrong.  I'll get this fixed up!

**Comment by SevereOverfl0w** [comment]
I'm still interested, yep!  I have a dracut module too I use to set the colours on startup which is nice for my eyes which don't do well with white-on-black.

**Comment by ericonr** [comment]
Maybe consider PRing the dracut module to the repo or something of the sort?

**Comment by SevereOverfl0w** [comment]
Yep, that's a follow up on my list! :)

**Pull request merged**